### ISSUE TITLE
Assets command really needs to tell me what circular dependency it is finding

### ISSUE THREAD

**Issue by Sammaye** [yiisoft/yii2#7539]
I keep getting:
```
Creating new bundle configuration...
Error: A circular dependency is detected for target 'widgets'.
```
But the problem is it doesn't actually tell me which one, even in debug mode it doesn't seem to.

**Comment by cebe** [comment]
showing which one exactly is quite hard because it may be over a long list of dependencies which then depend back on the first element. Not sure if it really helps if we implement a search strategy here to improve the error message. When seeing this messages you could normally go through the list of dependencies of 'widgets' and check if any of them depend back on 'widgets'.

**Comment by creocoder** [comment]
@cebe Probably such functionality when `YII_DEBUG` is `true` may be interesting.

**Comment by klimov-paul** [comment]
Issue resolved by commit 378937f
